sdk/program/src/syscalls/definitions.rs

```diff
@@ -1,5 +1,7 @@
 #[cfg(target_feature = "static-syscalls")]
 pub use solana_define_syscall::sys_hash;
+#[deprecated(since = "2.1.0", note = "Use `solana-msg::sol_log` instead.")]
+pub use solana_msg::sol_log;
```

Comment:
Looks like this replaced the re-export name from `sol_log_` to `sol_log` as noticed during CI for #3271, so I'm fixing that over there